test_pull_requests

```diff
@@ -124,12 +124,38 @@ NEEDS_REBASE_LABEL          = "needs-rebase"
 GITHUB_API_BASE_URL = 'https://api.github.com'
 GITHUB_BASE_URL = 'https://github.com'
 
+CONTENT_RUNNING = "Running"
+CONTENT_WAITING = "Waiting"
+CONTENT_EVALUATING = "Evaluating for testing"
+
+CONTENT_TO_STATE = {
+  CONTENT_EVALUATING => :evaluating,
+  CONTENT_RUNNING => :running,
+  CONTENT_WAITING => :waiting,
+  "SUCCESS" => :success,
+  "FAILURE" => :failure,
+  "ABORTED" => :aborted,
+  "UNSTABLE" => :unstable,
+  "NOT_BUILT" => :not_built
+}
+
+STATE_TO_CONTENT = CONTENT_TO_STATE.invert
+
+BOT_COMMENT_FIELDS = {
+  :build_url => :no_prefix,
+  :base_commit => 'Base Commit:',
+  :extended_tests => 'Extended Tests:',
+}
+
 # We use a saturating counter to get a more stable reading of mergeability from
 # the GitHub API for a pull request. The following two values are the rails at
 # which we saturate.
 NOT_MERGEABLE = 10
 MERGEABLE = 0
 
+# base sleep time for API retries
```

Comment:
We only use units of time with Ruby's built-in `sleep`, which only takes seconds. I don't think we really need to specify the units here.